src/emmy/expression/render.cljc

```diff
@@ -39,9 +36,12 @@
   simplifier negates by wrapping with `(* -1 ...)`. For rendering, we prefer to
   use a unary minus."}
   rewrite-negation
+
   (R/ruleset
    (* -1 ?x) => (u- ?x)
-   (* -1 ??x) => (u- (* ??x))))
+   (* -1.0 ?x) => (u- ?x)
+   (* -1 ??x) => (u- (* ??x))
```

Comment:
You could also write this pattern as `(* (? _ #{-1.0 -1}) ??x)`

Reply:
done